src/Panel/Page.php

```diff
@@ -194,4 +186,83 @@ public function route(): array
             'title'      => $page->title()->toString(),
         ];
     }
+
+    /**
+     * Returns the next page in defined navigation
+     *
+     * @return \Kirby\Cms\Collection
+     */
+    public function next()
```

Comment:
I think we need to consider names here since we also have `Model::prevnext()` - so to see what they actually do. Also since `next()` and `prev()` are so similar, maybe it could be one method that receives the direction as parameter.

Reply:
Firstly I'm thinking of changing about `prevnext()` method name. This method doesn't return previous or next collection. Only returns link and tooltip of current model, right?. May be `info` name more clear or something like that.

Reply:
Also I would like `prev()` and `next()` to be separate methods and to be consistent with what we use in other classes.

Reply:
Wouldn't it make sense to keep `prev()` and `next()` as separate methods and rename `prevnext()` into `navigation()`. Then it would be more in line with the blueprint settings. Just an idea.

Reply:
`navigation()` name more clear/accurate than `prevnext()` 👍